src/hero.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="/dist/output.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500;600;700;800;900&family=Roboto&display=swap" rel="stylesheet">
+    <title>hero component</title>
+</head>
+<body class="bg-back">
+    <article class="inline-flex space-x-151">
+        <p class="w-737 h-151 text-tx font-bold font-text text-2xl text-justify py-20 px-51.7 mx-auto">IEEE Computer Society
+        VIT Chennai's Docs</p>
+        <p class="w-628.59 h-165.64 text-tx text-3xl text-justify space-y-737 px-51.7 py-51.7 mx-auto">The starting point for your tech journey, Get started and explore you favorite domain in tech. Whether you're just starting out or you're an experienced developer looking to take your skills to the next level, our documentation has something for <strong>everyone</strong>.</p>
+    </article>
```

Comment:
Only this bit of code is required if you re-wrote this in JSX

To learn what JSX is:
- [JSX from the React Team](https://reactjs.org/docs/introducing-jsx.html)
- [JSX - A comprehensive guide](https://facebook.github.io/jsx/)